data/transactions/logic/evalStateful_test.go

```diff
@@ -1081,7 +1072,7 @@ byte 0x414c474f
 	testApp(t, text, now, "no such app")
 
 	// create the app and check the value from ApplicationArgs[0] (protocol.PaymentTx) does not exist
-	ledger.newApp(txn.Txn.Sender, 100, makeSchemas(0, 0, 0, 1))
+	ledger.newApp(txn.Txn.Sender, 100, basics.AppParams{})
```

Comment:
NumByteSlices used to be 1 and it is 0 now, why is it?

Reply:
The NumByteSlices was never used to check correctness of puts in these tests.  I only added it recently to unit test min_balance.  It definitely wasn't needed here.

While it might be worth making the testLedger check such things, I'm more interested in getting rid of the testLedger, and instead construct a "real" logicLedger, using a mock Balances object.  It would be more realistic and help me test the txn creation code.